File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        s_list = list(s)
        t_list = list(t)
        s_list.sort()
        t_list.sort()

        if ''.join(s_list) == ''.join(t_list):
            return True
        return False
        

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        List<String> sList = Arrays.asList(s.split(""));
        List<String> tList = Arrays.asList(t.split(""));
        Collections.sort(sList);
        Collections.sort(tList);

        if (sList.equals(tList))
            return true;
        return false;
    }
}
